README.md

```diff
@@ -22,6 +22,10 @@ The nightlies are also published to a F-Droid repository which you can add by sc
 
 or clicking [this link](https://jhass.github.io/insporation/fdroid/repo?fingerprint=EC792A58B39DF9FBB466FB100E30E3842F229FDBC6E28D32C417F6A5B30ECCAE) on your device that has F-Droid installed.
 
+#### IzzyOnDroid F-Droid repo
+
+The app is also available via the IzzyOnDroid repo [here](https://apt.izzysoft.de/fdroid/index/apk/jhass.eu.insporation). You have to add this repo to your F-Droid client first, you can do so by clicking this [link](https://apt.izzysoft.de/fdroid/repo?fingerprint=3BF0D6ABFEAE2F401707B6D966BE743BF0EEE49C2561B9BA39073711F628937A).
```

Comment:
Let's also include the notice that clicking the link needs to happen on the device, like above.

Reply:
Done, and I reworded the sentence to "a F-Droid client installed" since there is not only the official F-Droid client existing.